projects/packages/jetpack-mu-wpcom/src/features/launchpad/launchpad.php

```diff
@@ -680,6 +680,33 @@ function wpcom_launchpad_set_fse_next_steps_modal_hidden( $should_hide ) {
 	return update_option( 'wpcom_launchpad_config', $wpcom_launchpad_config );
 }
 
+/**
+ * Helper function to set the current active checklist in the navigator context.
+ *
+ * @param string $checklist_slug The slug of the launchpad task list to mark as active.
+ * @return bool Whether the option update succeeded.
+ */
+function wpcom_launchpad_set_current_active_checklist( $checklist_slug ) {
+	$wpcom_launchpad_config = get_option( 'wpcom_launchpad_config' );
+
+	$checklists = wpcom_launchpad_checklists()->get_all_task_lists();
+	if ( ! array_key_exists( $checklist_slug, $checklists ) ) {
+		return false;
+	}
```

Comment:
I think we should allow `$checklist_slug` to be `null`, and only run this logic if it's not null.
```suggestion
	if ( null !== $checklist_slug ) {
		$checklists = wpcom_launchpad_checklists()->get_all_task_lists();
		if ( ! array_key_exists( $checklist_slug, $checklists ) ) {
			return false;
		}
	}
```